osu.Game/Overlays/Settings/SettingsDropdown.cs

```diff
@@ -13,39 +13,23 @@ public class SettingsDropdown<T> : SettingsItem<T>
     {
         protected new OsuDropdown<T> Control => (OsuDropdown<T>)base.Control;
 
-        private IEnumerable<T> items = Enumerable.Empty<T>();
-
         public IEnumerable<T> Items
         {
-            get => items;
-            set
-            {
-                items = value;
-
-                if (Control != null)
-                    Control.Items = value;
-            }
+            get => Control.Items;
+            set => Control.Items = value;
         }
 
-        private IBindableList<T> itemSource;
-
         public IBindableList<T> ItemSource
```

Comment:
Probably needs an xmldoc note that this will no longer handle the bindable reference storing (unlike `Bindable` in `SettingsItem`).

Reply:
Why is that relevant? The underlying control will.

Reply:
As discussed, this is one case it isn't currently. Should be fixed framework side. Also `SettingsItem.Bindable` local storage can be removed.

Reply:
Probably best to keep the one inside SettingsItem, as it's possible for `controlWithCurrent` to be null and for `ShowsDefaultIndicator` to be false.